### PR TITLE
Omertsur ocp311 compatability

### PR DESCRIPTION
**Pull request by omertsur**
- Added k8s 1.11 or older supported CRD template.
- Edited the "getting started" section of the README for legacy support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,30 @@
...
 kubectl create namespace observability
+```
+2) Load the CRD based on your version:
+- Kubernetes 1.12+:
+```
+kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml
+```
+- Kubernetes 1.11-: 
+```
+kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd_ocp311.yaml
```
Comment (objectiser) [inline README.md:20]: Could you change the CRD filename to end `k8s1.11.yaml`? as listed under the k8s 1.11 header

File: README.md [hunk 2]
```diff
@@ -6,18 +6,30 @@
...
+1) Create the namespace: 
 ```
 kubectl create namespace observability
+```
+2) Load the CRD based on your version:
+- Kubernetes 1.12+:
+```
+kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml
+```
+- Kubernetes 1.11-:
```
Comment (objectiser) [inline README.md:18]: Possibly add  ` / OpenShift 3.11`?

File: README.md [hunk 3]
```diff
@@ -6,18 +6,30 @@
...
+kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml
+```
+- Kubernetes 1.11-: 
+```
+kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd_ocp311.yaml
+```
+
+3) To install the operator, run:
+```
 kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml
```
Comment (objectiser) [inline README.md:25]: This line would need to be removed, as now included in the previous section.

File: README.md (hunk 2 above)
Comment (omertsur) [reply README.md:18]: I refrained from talking about OCP beacuse it isn't mentioned anywhere here for some reason, I actually use Openshift personally,

File: README.md (hunk 3 above)
Comment (omertsur) [reply README.md:25]: My bad, missed removing it

**Comment by omertsur** [comment]
I created the commit from the web interface (haven't even cloned the repo to my computer). Is there any way to sign it that way?

**Comment by objectiser** [comment]
@omertsur Not that I am aware - if it was a single commit then you may be able to just add the "Signed by...." text, but as each commit needs to be signed, and you cannot edit those without cloning.

**Comment by jpkrohling** [comment]
I'm not sure we should have this in the operator repository, as it might give the (false) impression that support 3.11 in some way.

**Comment by jpkrohling** [comment]
I'm closing this as I'm still not sure we want to support such an ancient version of OpenShift.

**Pull request closed without merging**